code/game/objects/items/food/soup.dm

```diff
@@ -149,12 +149,12 @@
 	tastes = list("tomato" = 1, "squirming" = 1)
 	foodtypes = MEAT | GROSS
 
-/obj/item/food/soup/milo
-	name = "milosoup"
+/obj/item/food/soup/miso
+	name = "misosoup"
 	desc = "The universes best soup! Yum!!!"
-	icon_state = "milosoup"
+	icon_state = "misosoup"
 	food_reagents = list(/datum/reagent/consumable/nutriment = 4, /datum/reagent/consumable/nutriment/protein = 8, /datum/reagent/water = 5, /datum/reagent/consumable/nutriment/vitamin = 4)
-	tastes = list("milo" = 1) // wtf is milo //i don't know either but hey i guess japanese tourists will order it
+	tastes = list("miso" = 1) // wtf is milo //i don't know either but hey i guess japanese tourists will order it. //I carshalash the cringe fixed it
```

Comment:
remove all these comments, comments should say what code is, not what it used to be

Reply:
keep the meme alive

Reply:
he will not divide us